src/Header/Account/Account.jsx

```diff
@@ -148,16 +175,18 @@ export default class Account extends Component {
 			</div>
 		) : (
 			<a
-				href={signinUrl}
+				href={signInLink.value}
 				className={styles.button}
 				onClick={this.handleMenuItemClick}
 			>
-				Sign in
+				{signInLink.key}
 			</a>
 		);
 	}
 }
 
+Account.idamProvider = "idam";
```

Comment:
I wonder if this is better as an object like https://github.com/nhsevidence/nice-design-system/blob/v1/packages/nds-panel/src/Panel.js#L31-L35. Then we can use it within the `provider` propType def like:

```
provider: PropTypes.oneOf([Account.providers.niceAccounts, Account.providers.idam]),
```